build.gradle

```diff
@@ -17,21 +17,16 @@
  * along with the Cyface SDK for Android. If not, see <http://www.gnu.org/licenses/>.
  */
 /**
- * Gradle's top-level build file which defines the repositories to pull the dependencies from
- * and the default dependency versions used in most modules.
```

Comment:
I tried to clean up the build.gradle files and make them all similar (of all android projects) so that we can identify required differences between the build.gradle files easier.